USAGE.md

```diff
@@ -19,77 +19,358 @@ Now `make install` only supports Linux Ubuntu, CentOS and Mac OS X. If you use o
+      - --etcd-listen-peer-url=http://0.0.0.0:2380
+      - --etcd-listen-client-url=http://0.0.0.0:2379 
+
+    privileged: true
+
+  pd2:
+    image: pingcap/pd
+    ports:
+      - "1234"
+      - "9090"
+      - "2379"
+      - "2380"
+
+    command:
+      - --addr=0.0.0.0:1234
+      - --advertise-addr=pd2:1234
+      - --http-addr=0.0.0.0:9090
+      - --cluster-id=1
+      - --etcd-name=pd2
+      - --etcd-advertise-client-url=http://pd2:2379
+      - --etcd-advertise-peer-url=http://pd2:2380
+      - --etcd-initial-cluster=pd1=http://pd1:2380,pd2=http://pd2:2380,pd3=http://pd3:2380
+      - --etcd-listen-peer-url=http://0.0.0.0:2380 
+      - --etcd-listen-client-url=http://0.0.0.0:2379 
+
+    privileged: true
+
+  pd3:
+    image: pingcap/pd
+    ports:
+      - "1234"
+      - "9090"
+      - "2379"
+      - "2380"
+
+    command:
+      - --addr=0.0.0.0:1234
+      - --advertise-addr=pd3:1234
+      - --http-addr=0.0.0.0:9090
+      - --cluster-id=1
+      - --etcd-name=pd3
+      - --etcd-advertise-client-url=http://pd3:2379 
+      - --etcd-advertise-peer-url=http://pd3:2380 
+      - --etcd-initial-cluster=pd1=http://pd1:2380,pd2=http://pd2:2380,pd3=http://pd3:2380 
+      - --etcd-listen-peer-url=http://0.0.0.0:2380 
+      - --etcd-listen-client-url=http://0.0.0.0:2379 
+
+    privileged: true
+
+  tikv1:
+    image: pingcap/tikv
+    ports:
+      - "20160"
+
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv1:20160
+      - --cluster-id=1
+      - --dsn=raftkv
+      - --store=/var/tikv
+      - --etcd=pd1:2379,pd2:2379,pd3:2379
+
+    depends_on:
+      - "pd1"
+      - "pd2"
+      - "pd3"
+
+    entrypoint: /tikv-server
+
+    privileged: true
+
+  tikv2:
+    image: pingcap/tikv
+    ports:
+      - "20160"
+
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv2:20160
+      - --cluster-id=1
+      - --dsn=raftkv
+      - --store=/var/tikv
+      - --etcd=pd1:2379,pd2:2379,pd3:2379
+
+    depends_on:
+      - "pd1"
+      - "pd2"
+      - "pd3"
+
+    entrypoint: /tikv-server
+
+    privileged: true
+
+  tikv3:
+    image: pingcap/tikv
+    ports:
+      - "20160"
+
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv3:20160
+      - --cluster-id=1
+      - --dsn=raftkv
+      - --store=/var/tikv
+      - --etcd=pd1:2379,pd2:2379,pd3:2379
+
+    depends_on:
+      - "pd1"
+      - "pd2"
+      - "pd3"
+
+    entrypoint: /tikv-server
+
+    privileged: true
+
+  tidb:
+    image: pingcap/tidb
+    ports:
+      - "4000"
+
+    command:
+      - --store=tikv 
+      - --path=pd1:2379,pd2:2379,pd3:2379/pd?cluster=1
+      - -L=info
+
+    depends_on:
+      - "tikv1"
+      - "tikv2"
+      - "tikv3"
+
+    privileged: true
+    
+```
+
++ Use `docker-compose up -d` to create and start the cluster. 
++ Use `docker-compose port tidb 4000`, output may be `0.0.0.0:32966`, so the real TiDB host port is `32966`.
```

Comment:
++ Use `docker-compose port tidb 4000` to print the TiDB host port. For example, if the output is `0.0.0.0:32966`, the TiDB host port is `32966`.